### ISSUE TITLE
remove mkl blas lincense

### ISSUE THREAD

**Issue by roywei** [apache/incubator-mxnet#15544]
According to @general discussion, we should remove MKL BLAS lincense if we are not shipping it with our source release.

https://mail-archives.apache.org/mod_mbox/incubator-general/201907.mbox/%3C29E387FB-238F-4FC2-A41D-E4AB99AF4911%40classsoftware.com%3E

Currently we added it in license file here: https://github.com/apache/incubator-mxnet/blob/master/LICENSE#L624

cc @pengzhao-intel

**Comment by roywei** [comment]
@mxnet-label-bot add [Licenses]

**Comment by pengzhao-intel** [comment]
Yes, I think it makes sense to remove MKL license because we don't distribute it.

@roywei would you mind file a PR to remove it?

**Comment by pengzhao-intel** [comment]
cc @TaoLv 

@tprimak @vpirogov 
Could you help to confirm below two points?
What's the license of MKLML in the MKL-DNN?
Is it OK to remove MKL license and only leave Apache 2.0 for MKL-DNN?

**Comment by paulk-asert** [comment]
It isn't to do with "full MKL package being an option" per se, although related. There is a file that is downloaded as part of the build process that has the "Intel® Simplified license". The file is downloaded under {{external}} and then copied into {{build/install}}. However, there is no source file in your source tarball that has such a license, hence it should not be part of the LICENSE file.

**Comment by paulk-asert** [comment]
I should also note that while removing the section from the source LICENSE file fixes the source tarball, you still seem to have issues with other artefacts, e.g. mxnet-full_2.11-linux-x86_64-cpu-1.5.0-SNAPSHOT.jar (as an example) contains the affected .so files and so should mention that license within it's META_INF/LICENSE file which it does not (nor any of the other compatible licenses). Of course, then the issues which @jmclean mentioned wrt category-x apply to that artefact. The general strategy I have seen used in such cases is to have the respective .so files come in via an optional non-Apache artefact and be automatically added to library path as needed.

**Comment by samskalicky** [comment]
> Yes, I think it makes sense to remove MKL license because full MKL package is an option.
> 
> @roywei would you mind file a PR to remove it?

@roywei Did you remove this, or did we just modify the license appropriately? Can you summarize what is left to do to close this issue?

**Comment by roywei** [comment]
@samskalicky I think this can be closed when we integrate with mkldnn v1.0, which is still in a branch https://github.com/apache/incubator-mxnet/tree/mkldnn-v1.0

**Comment by TaoLv** [comment]
@roywei @samskalicky I think this has been fixed on master branch. I'm closing this issue. Feel free to reopen if any concern. Thanks.
